_submissions/round-12/1/2015-02-25-functions-in-programming-languages.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2015-02-25
+round: Round 12
+title: Functions in Programming Languages
+author: Kai Yang
+permalink: /2015/02/functions-in-programming-languages/
+tags:
+  - Concept Map
+  - Function
+  - Programming Language
+---
+SWC link:
+http://swcarpentry.github.io/python-novice-inflammation/04-func.html
+
+This concept map is intended to illustrate functions in programming languages, including the design and use of functions.
+
+Please review and download at 
+[Functions in Programming Languages](http://imgur.com/gWC7JpI)
```

Comment:
The link will need to be `http://i.imgur.com/gWC7JpI.jpg` to display the image - using `http://i.imgur.com/gWC7JpI` on its own is just a link to the Imgur page.